repository/ConfigurationOfOSSubprocess.package/ConfigurationOfOSSubprocess.class/instance/stable..st

```diff
@@ -2,4 +2,6 @@ symbolic versions
 stable: spec
 	<symbolicVersion: #stable>
 	
+	spec for: #'common' version: '0.2.4'.
```

Comment:
I agree with @ThierryGoubier  here too, so I also prefer adding the #'pharo6.x'  as you did finally instead of #'common'.